apps/openmw/mwmechanics/spellcasting.cpp

```diff
@@ -433,22 +426,28 @@ namespace MWMechanics
                                     "meshes\\" + reflectStatic->mModel, ESM::MagicEffect::Reflect, false, "");
                         reflectedEffects.mList.push_back(*effectIt);
                         magnitudeMult = 0;
+                        break;
```

Comment:
Yes. Just to be sure, I tested in the original game and confirmed that some effects of a spell can be absorbed or reflected while others aren't.